src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -770,7 +770,10 @@ messages.etmsg.desktop.telemetryImportError = Unable to import file, try again l
 messages.fmsg.m0001 = Could not fetch enroled courses, try again later
 messages.fmsg.m0002 = Could not fetch other courses, try again later...
 messages.fmsg.m0003 = Unable to get batch schedule details
-messages.fmsg.m0004 = Could not fetch date, try again later
+
+# Don't change this messages.fmsg.m0004
```

Comment:
no this has been used in the batch create and all please add a new variable for data please dont change date and data